frontend/src/components/applyForm/ApplyFormMessage.tsx

```diff
@@ -0,0 +1,55 @@
+import { useTranslations } from "next-intl";
+import { Alert } from "@trussworks/react-uswds";
+
+import { FormValidationWarning } from "./types";
+
+export const ApplyFormMessage = ({
+  error,
+  validationWarnings,
+  submitted,
+}: {
+  error: boolean;
+  validationWarnings: FormValidationWarning[] | null;
+  submitted: boolean;
+}) => {
+  const t = useTranslations("Application.applyForm");
+  const errorMessage = t.rich("errorMessage", {
+    "email-link": (content) => (
+      <a href="mailto:simpler@grants.gov">{content}</a>
+    ),
+    p: (content) => <p>{content}</p>,
```

Comment:
got an error about `<p> can't be a descendant of <p> `related to this. Maybe this `<p>` could just be a div?